pkg/apis/deployment/v1alpha/server_group_spec.go

```diff
@@ -110,8 +131,14 @@ func (s ServerGroupSpec) Validate(group ServerGroup, used bool, mode DeploymentM
 				minCount = 2
 			}
 		}
+		if s.GetCount() < s.GetMinCount() {
```

Comment:
Forgot earlier, check minCount <= maxCount

Reply:
Yeah I can add that check.